pandas/tests/arrays/string_/test_string.py

```diff
@@ -325,32 +327,29 @@ def test_from_sequence_no_mutate(copy, cls, request):
     tm.assert_numpy_array_equal(nan_arr, expected)
 
 
-def test_astype_int(dtype, request):
-    if dtype == "arrow_string":
-        reason = "Cannot interpret 'Int64Dtype()' as a data type"
-        mark = pytest.mark.xfail(raises=TypeError, reason=reason)
-        request.node.add_marker(mark)
+def test_astype_int(dtype):
+    arr = pd.array(["1", "2", "3"], dtype=dtype)
+    result = arr.astype("int64")
+    expected = np.array([1, 2, 3], dtype="int64")
+    tm.assert_numpy_array_equal(result, expected)
+
+    arr = pd.array(["1", pd.NA, "3"], dtype=dtype)
```

Comment:
prob best in a dedicate _errors test

Reply:
sure. done in https://github.com/simonjayhawkins/pandas/commit/e1577d4c6a497e8212d5a94809b0a3ffeac9e66c

will open a PR with other follow-ups